src/android/ImageResizer.java

```diff
@@ -62,7 +66,7 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
         callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.OK, scaledFile.toString()));
         return true;
       } else {
-        callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.ERROR));
+          callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.ERROR));
```

Comment:
I don't work at this company anymore and don't have the rights to merge anything... This pull requests looks good except the 4 spaces everywhere O.o

Maybe @tiff can help?